### PR TITLE
Updated Lecture 2. Fixed links/images. Added Description for running …

### PR DESCRIPTION
**Pull request by k1p1**
…the Engine from Epic Launcher.

### CODE REVIEW

File: lectures/02.intro_ue4.md [hunk 1]
```diff
@@ -547,7 +551,7 @@ placed into one of these categories<sup>1</sup>:
 
 --- VERTICAL SLIDE ---
 
-As of 4.15.0, there are **265** modules!
```
Comment (nikoladimitroff) [inline lectures/02.intro_ue4.md:550]: How about 4.23?

File: lectures/02.intro_ue4.md [hunk 2]
```diff
@@ -566,6 +570,8 @@ grep -r "public IModuleInterface" /Engine/Source/ | wc -l
 * There are few really important classes
     - `UObject` is the father of all; provides reflection and garbage collection
     - `AActor` is the parent of everything placeable in the world. Derived from `UObject`
+	- `APawn` is the parent of everything possessable by players or AI. Derived from `AActor`
+	- `AController` can possess a Pawn to control its actions. Derived from `UObject`
```
Comment (nikoladimitroff) [inline lectures/02.intro_ue4.md:574]: I believe we deliberately left these out in order to not confuse people. They are better explained in architecture UE4

**Review by nikoladimitroff — APPROVED** [review]

File: lectures/02.intro_ue4.md (hunk 2 above)
Comment (k1p1) [reply lectures/02.intro_ue4.md:574]: I can't find anything about APawn and AController. Also we can explain it in more details.

File: lectures/02.intro_ue4.md (hunk 1 above)
Comment (k1p1) [reply lectures/02.intro_ue4.md:550]: I'll update it when I download 4.23

**Pull request merged**